### PR TITLE
Use folio community script for DAGs to run and email script logs

### PR DESCRIPTION
**Pull request by jgreben**
- DAGs for Lane, Law and SUL.
- Wrapper function for community fix_encumbrances.py script.
- Email stdout saved to log with shared dir created for log dir.

### CODE REVIEW

File: libsys_airflow/dags/folio_finance/lane_fix_encumbrances.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+default_args = {
+    "owner": "libsys",
+    "depends_on_past": False,
+    "email_on_failure": True,
+    "email_on_retry": False,
+    "retries": 1,
+    "retry_delay": timedelta(minutes=1),
+}
+
+FY_CODE = Variable.get("FISCAL_YEAR_CODE_LANE", "LANE2025")
```
Comment (jermnelson) [inline libsys_airflow/dags/folio_finance/lane_fix_encumbrances.py:25]: We may want to add a section to the FYRO checklist to update these variables.

**Review by jermnelson — APPROVED** [review]
Approving with a comment to add setting the FY Variables to the FYRO checklist.

File: libsys_airflow/dags/folio_finance/lane_fix_encumbrances.py (hunk 1 above)
Comment (jgreben) [reply libsys_airflow/dags/folio_finance/lane_fix_encumbrances.py:25]: https://docs.google.com/spreadsheets/d/1T1G456G5jiq1G1sHuj-ehJ0u2ukPeTOnnMcpbAv29C0/edit?gid=768670530#gid=768670530

File: libsys_airflow/plugins/folio/encumbrances/email.py [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+
+def email_log(**kwargs):
+    library = kwargs.get("library")
+    log_file = kwargs.get("log_file")
+    devs_email = Variable.get("EMAIL_DEVS")
+
+    to_addresses = [
+        devs_email,
+    ]
```
[the remark this answers is not in the document]
Comment (jgreben) [reply libsys_airflow/plugins/folio/encumbrances/email.py]: I added the (tbd) addresses for lane, law, and sul.

**Pull request merged**